kratos.gid/apps/FluidDEM/xml/Main.spd

```diff
@@ -5,6 +5,7 @@
     <include n="AnalysisType"        active="1" path="apps/Fluid/xml/AnalysisType.spd"/>
     <include n="FluidParts"          active="1" path="apps/Fluid/xml/Parts.spd"/>
     <include n="InitialConditions"   active="1" path="apps/Fluid/xml/InitialConditions.spd"/>
+    <include n="ManufacturedConditions" active="1" path="apps/FluidDEM/xml/ManufacturedConditions.spd"/>
```

Comment:
Are you sure this is of general interest? I mean, if it's for testing, it should be in the tests, but not in the GUI. No normal user should be interested in using ManufacturedConditions.

Reply:
I think could be useful. If you know the exact solution of a variable you can impose it by using that condition. In the tree is set as "Exact Conditions"